test/unit/services/TagPresenter.test.js

```diff
@@ -0,0 +1,96 @@
+var root = require('root-path')
+var setup = require(root('test/setup'))
+var factories = require(root('test/setup/factories'))
+import moment from 'moment'
+import { find } from 'lodash/fp'
+import { fetchAndPresentForCommunity } from '../../../api/services/TagPresenter.js'
+describe('fetchAndPresentForCommunity', () => {
+  var fixtures
```

Comment:
let's drop the `fixtures` umbrella variable and just have individual variables for each object -- way less wordy

e.g.:
```js
var u1, u2, u3, u4
;[ u1, u2, u3, u4 ] = times(4, () => factories.user())
// ...
Promise.map([u1, u2, u3, u4], x => x.save())
```

likewise maybe you can create a list of the user/tag pairs you need to create CommunityTags for and forEach them instead of writing out lines 25 through 51